doc.css

```diff
@@ -0,0 +1,66 @@
+h4.name {
```

Comment:
I noticed the "home" link on top of the navbar is still purple.

Otherwise, very nice color, a lot better, thanks!